lib/generators/onesie/templates/task.rb

```diff
@@ -7,7 +7,7 @@ class <%= class_name %> < Onesie::Task
       manual_task enabled: false
 
       def run
-        # Write your Onesie Task here
+<%= run_contents.chomp.indent(8) %>
```

Comment:
Does this line not work if it is intended?

Reply:
Do you see a template in which consumers of the gem _do not_ want the output to be formatted properly?

Reply:
No, it should be formatted correctly. I was wondering why it was required to call the `.indent` method. I'm assuming it has to do with the template parsing.

Reply:
I don't know if required is the right term here. From a formatting perspective, if we don't include the indent, a onesie generated via template will appear as:

```ruby
# frozen_string_literal: true
module Onesie
  module Tasks
    class <%= class_name %> < Onesie::Task
      allowed_environments :all
      manual_task enabled: false
      def run
template here      
      end
    end
  end
end
```

Which will either make engineer's eyes twitch (or rubocop 😂 )